torch/nn/__init__.py

```diff
@@ -1,11 +1,11 @@
 # mypy: allow-untyped-defs
-from torch.nn.parameter import (
+from torch.nn.parameter import (  # usort: skip
     Buffer as Buffer,
     Parameter as Parameter,
     UninitializedBuffer as UninitializedBuffer,
     UninitializedParameter as UninitializedParameter,
 )
```

Comment:
`torch.nn.parameter` needs to be imported first. We have a circular dependency between `torch.nn.parameter` and `torch.autograd`. See also:

- #128609